src/_P016_IR.ino

```diff
@@ -96,9 +86,13 @@ decode_results results;
 // Set lower if you are sure your setup is working, but it doesn't see messages
 // from your device. (e.g. Other IR remotes work.)
 // NOTE: Set this value very high to effectively turn off UNKNOWN detection.
-#define MIN_UNKNOWN_SIZE 12
+const uint16_t kMinUnknownSize = 12;
 // ==================== end of TUNEABLE PARAMETERS ====================
 
+
+IRrecv *irReceiver;
```

Comment:
Ok I am done for today, getting tired and start making mistakes.
Can you please direct me in how to send strings back as a value to ESPeasy so we can forward them to the condroler (mqtt)? I want to forward Ir received codes as mqtt.

Reply:
What strings do you want to return?
Can you give an example of the type of string?

Reply:
The first one is the IR code that was received, like "NEC,566C8E" and the second is the IR2 code that was received if the encoding was sucessful. Perhaps also a third string witch will be the extended description given by the Ir library in case the IR code is comming from an AC unit

Examples:
First String: "GREE,0800205014200050" or "Unknown" if the decoding was unsucessfull
Second String: IR2, or "IR2,Unknown" if the decoding was unsucessfull
Thired String: "Power: Off, Mode: 0 (AUTO), Temp: 16C, Fan: 0 (AUTO), Turbo: Off, XFan: Off, Light: On, Sleep: Off, Swing Vertical Mode: Manual"
 or nothing if this was not a message from an AC

Reply:
I think that's a bit to complex to "just add it", so maybe you could add an issue for it?
That will allow for some discussion before implementing it.